docs/faq.rst

```diff
@@ -309,7 +309,7 @@ This will create a suffix for the node location for example:
    us-east1_my_dc
 
 
-The problem may also arise if you are using some :code:`DC-aware snitch`, e.g. :code:`Ec2MultiRegionSnitch`, and a :code:`SimpleStrategy` in a multi-DC cluster.
+The problem may also arise if you are using some :code:`DC-aware snitch`, e.g. :code:`Ec2MultiRegionSnitch`, and a :code:`NetworkTopologyStrategy` in a multi-DC cluster.
```

Comment:
This sentence says that there's some problem when combining these configuration options, but does the problem still occur if we use NetworkTopologyStrategy instead of SimpleStrategy? The former already accounts for multi-DC setup. Bottom line is: perhaps SimpleStrategy should stay, just an additional comment to user NetworkTopology instead could suffice.